test/tests/storage/storage_test.go

```diff
@@ -17,7 +17,7 @@ func TestCreateBucket(t *testing.T) {
 	it, _ := integration.NewTest(t, 30*time.Second)
 	defer it.Done()
 
-	rsa, err := it.Instrument(integration.ComponentWorkspaceDaemon, "daemon")
+	rsa, err := it.Instrument(integration.ComponentWorkspaceDaemon, "daemon", integration.WithContainer("ws-daemon"))
```

Comment:
maybe we should pull this `WithContainer` bit into the `Instrument` function. It already sees which component we want to instrument, hence could know how the container is called (basically not `kube-rbac-proxy`).